src/Microsoft.ML.Data/TrainContext.cs

```diff
@@ -331,7 +331,7 @@ internal ClusteringTrainers(ClusteringContext ctx)
                 Host.CheckNonEmpty(features, nameof(features), "The features column name should be non-empty, if provided, if you want to calculate the Dbi metric.");
 
             if (label != null)
-                Host.CheckNonEmpty(label, nameof(label), "The features column name should be non-empty, if provided, if you want to calculate the Nmi metric.");
+                Host.CheckNonEmpty(label, nameof(label), "The label column name should be non-empty, if provided, if you want to calculate the Nmi metric.");
```

Comment:
>The label column name should be non-empty, if provided, if you want to calculate the Nmi metric [](start = 58, length = 95)

"To calculate the NMI metric,  provide a value for the the optional label column".